src/objects/core/admin.py

```diff
@@ -13,6 +18,32 @@ class ObjectTypeAdmin(admin.ModelAdmin):
     )
     readonly_fields = ("_name",)
 
+    def get_urls(self):
+        urls = super().get_urls()
+        my_urls = [
+            path(
+                "<int:objecttype_id>/versions/",
+                self.admin_site.admin_view(self.versions_view),
+            )
+        ]
+        return my_urls + urls
+
+    def versions_view(self, request, objecttype_id):
+        versions = {
+            "count": 0,
+            "next": None,
+            "previous": None,
+            "results": [],
+        }
+        if objecttype := self.get_object(request, objecttype_id):
+            client = build_client(objecttype.service)
+            try:
+                response = client.get(objecttype.versions_url)
+                versions = response.json()
+            except (requests.RequestException, requests.JSONDecodeError):
+                pass
```

Comment:
Let's at least emit a log here
```suggestion
                logger.exception("Something went wrong while fetching objecttype versions")
```